### PR TITLE
Issue 153 - Fix Show Descriptions, Hosts, Links

### PR DESCRIPTION
**Pull request by harriseldon**
Updated the shows to match the existing site. I tried to add links with the links header. I am hoping that formatting them better becomes part of the hugo/css since they show up as plain links now.

### CODE REVIEW

File: content/show/linux-action-news/_index.md [hunk 1]
```diff
@@ -15,14 +15,20 @@ header_image = "/images/shows/linux-action-news.png"
...
-<li><a href="https://linuxactionnews.com">Show Notes</a></li>
-<li><a href="https://www.jupiterbroadcasting.com/contact/">E-Mail</a></li>
-<li><a href="https://twitter.com/linuxactionnews">Twitter</a></li>
-</ul>
+[links.twitter]
+  url = "https://twitter.com/linuxactionnews"
+[links.email]
+  url = "https://www.jupiterbroadcasting.com/contact/"  
+[links.reddit]
+  url = "https://www.reddit.com/r/coderradio"
```
Comment (gerbrent) [inline content/show/linux-action-news/_index.md:25]: coder radio reddit eh?
Hmm... not seeing this one on LAN.com.. 
Could you remove this entry?

File: content/show/linux-action-news/_index.md [hunk 2]
```diff
@@ -15,14 +15,20 @@ header_image = "/images/shows/linux-action-news.png"
...
-<li><a href="https://linuxactionnews.com/hosts/joe">Joe Ressington</a></li>
-<li><a href="https://linuxactionnews.com/hosts/chris">Chris Fisher</a></li>
-<li><a href="https://linuxactionnews.com">Show Notes</a></li>
-<li><a href="https://www.jupiterbroadcasting.com/contact/">E-Mail</a></li>
-<li><a href="https://twitter.com/linuxactionnews">Twitter</a></li>
-</ul>
+[links.twitter]
+  url = "https://twitter.com/linuxactionnews"
+[links.email]
+  url = "https://www.jupiterbroadcasting.com/contact/"
```
Comment (gerbrent) [inline content/show/linux-action-news/_index.md:23]: LAN.com lists https://linuxactionnews.com/contact ?

File: content/show/coder-radio/_index.md [hunk 3]
```diff
@@ -19,11 +19,14 @@ podverse_podcast_id = "ZXd_1Ojd9"
...
   url = "https://twitter.com/coderradioshow"
 [links.email]
-  url = "https://www.jupiterbroadcasting.com/contact/"  
+  url = "https://coder.show/contact"  
 [links.reddit]
   url = "https://www.reddit.com/r/coderradio"  
 [links.shownotes]
   url = "https://coder.show/"  
+[links.youtube]
+  url="https://www.youtube.com/watch?v=QQwdoS__pss&list=PLCB9545FD2C26E547"
```
Comment (gerbrent) [inline content/show/coder-radio/_index.md:28]: seems this link is formatted toward a specific video w playlist, not a playlist itself.... great upgrade idea though from just the standard JB YT link

get the playlist URL then we're 🎉 !

File: content/show/linux-unplugged/_index.md [hunk 4]
```diff
@@ -13,15 +13,30 @@ header_image = "/images/shows/linux-unplugged.png"
...
+  url="https://matrix.to/#/+lup:jupiterbroadcasting.com"
+
+[links.mumble]
+  url="https://linuxunplugged.com/mumble"
+
+[links.blog]
+  url="https://linuxunplugged.com/articles" 
+
+[links.youtube]
+  url="https://www.youtube.com/watch?v=CgTyhkI_qkE&list=PLUW3LUwQvegxVckkyK5dyyARdfjwGY-cw"
```
Comment (gerbrent) [inline content/show/linux-unplugged/_index.md:39]: same here re playlist itself vs video+playlist

should look more like you did in LAN: https://www.youtube.com/playlist?list=xxxxxxxx

File: content/show/self-hosted/_index.md [hunk 5]
```diff
@@ -15,6 +15,31 @@ header_image = "/images/shows/self-hosted.png"
...
+  url="https://wiki.selfhosted.show/"
+
+[links.email]
+  url="https://selfhosted.show/contact"
+
+[links.shownotes]
+  url="https://selfhosted.show"
+
+[links.youtube]
+  url="https://www.youtube.com/watch?v=bWLkoDU4OT0&list=PLUW3LUwQvegxit4XMxUNW3qrRFmgP_aaT"
```
Comment (gerbrent) [inline content/show/self-hosted/_index.md:38]: same here re YT video vs playlist

**Review by gerbrent — CHANGES_REQUESTED** [review]
Overall great stuff! Ahh.... that's better.

Just a few inconsistencies and adjustment to make, then we're golden.
Thanks for helping out!

File: content/show/linux-action-news/_index.md (hunk 1 above)
Comment (harriseldon) [reply content/show/linux-action-news/_index.md]: Good catch. I updated it.

File: content/show/coder-radio/_index.md (hunk 3 above)
Comment (harriseldon) [reply content/show/coder-radio/_index.md]: I did not realize there was a separate list url. Thanks.

**Review by gerbrent — APPROVED** [review]

**Pull request merged**